### DISCUSSION TITLE
[Dashboard] Refine Dashboard Capabilities: add column title Applies to and Operations

### DISCUSSION THREAD

**Discussion by zzxwill** [kubevela/kubevela#817] [General] [answered]
- Add column title `Applies to` and `Operations`
![image](https://user-images.githubusercontent.com/2805315/103117672-71a3ae80-46a6-11eb-99c0-6edfe5645772.png)

**Comment by sunny0826** [comment]
This is a list component, not a table
do not have column title

**Reply by zzxwill** [reply]
What does `Type` mean in the title field?

**Comment by zzxwill** [comment]
Is there any ways to delete `applies to` and move it to the very first line?

**Comment by sunny0826** [comment, marked as answer]
It may be necessary to try a better method of presentation.
